easy-random-core/src/main/java/org/jeasy/random/RandomizationContext.java

```diff
@@ -105,7 +108,7 @@ boolean hasExceededRandomizationDepth() {
     }
 
     private int nextInt(int startInclusive, int endExclusive) {
```

Comment:
This method can be removed and replace the line where it is called with:

```java
int randomIndex = actualPoolSize > 1 ? random.nextInt(actualPoolSize) : 0;
```

Reply:
done